server/graphql/types/src/types/program/encounter.rs

```diff
@@ -299,43 +349,92 @@ impl EncounterNode {
         &self,
         ctx: &Context<'_>,
         at: Option<DateTime<Utc>>,
-        filter: Option<EncounterEventFilterInput>,
-    ) -> Result<Vec<ProgramEventNode>> {
+        filter: Option<ActiveEncounterEventFilterInput>,
+        page: Option<PaginationInput>,
+        sort: Option<ProgramEventSortInput>,
+    ) -> Result<ProgramEventResponse> {
```

Comment:
The `active_program_events` field usually only returned one event for which reason a simple Vec was returned. However, the `program_events` field might return more events and have page and sort options. For consistency return the connector and add page and sort options.